invenio_rdm_records/theme/templates/invenio_rdm_records/coming_soon_page.html

```diff
@@ -0,0 +1,48 @@
+{#
+    Copyright (C) 2020 CERN.
+    Copyright (C) 2020 Northwestern University.
+  
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+#}
+  
+{%- extends config.BASE_TEMPLATE %}
+
+{%- block css %}
+{{ webpack['invenio-app-rdm-theme.css'] }}
+{{ webpack['invenio-rdm-records-theme.css'] }}
+{%- endblock %}
+
+{%- block page_body %}
+<div class="container">
+    <br>
+    <span class="section-title">
+        <h1>Keep calm and InvenioRDM ;)</h1>
+        <br>
+        <br>
+        <img width="600" align="center" src="{{ url_for('static', filename='images/coming-soon.svg')}}" />
+    </span>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
```

Comment:
Made an issue for "sticky footer": https://github.com/inveniosoftware/invenio-app-rdm/issues/81. I've done it in the past for our instance.

Reply:
It's true, tho also the coming soon is temporary. But in any case it would be nice to have the footer sticky at the bottom in case of few results in search or so :100:

Reply:
@fenekku @ppanero perfect because I was about to make my footer sticky :) So I will leave it for another day or should I do it?

Reply:
Leave it for later, unless it takes you just a few seconds :)